examples/overlay-warnings/README.md

```diff
@@ -0,0 +1,11 @@
+# Overlay
+
+```shell
+node ../../bin/webpack-dev-server.js --open
+```
+
+## What should happen
+
+The script should open the browser and show a heading with "Example: overlay with warnings".
+
+In `app.js`, make a syntax error. The page should now refresh and show a full screen error overlay that shows the syntax error.
```

Comment:
The problem is that those two files resolve fine on macOS. It would fail only on Windows and so on.